### PR TITLE
Tools: Enhanced screenshots

### PR DESCRIPTION
**Pull request by loremattei**
This PR adds two scripts for the generation of  screenshots with a background and a promo message.
The script starts from standard screenshots, resizes the image, paste it on top of a background and adds a custom message.
The scripts support the download of translated promo messages from GlotPress.

### Prerequisites
The script expects the standard screenshot to be available in the `Scripts/fastlane/screenshots` folder. If you don't have them, you can generate a set with the `fastlane snapshot` tool.

### Screenshot generation
1. from  `Scripts` folder, run `bundle exec fastlane download_promo_strings` in order to download the translated strings;
2. from  `Scripts` folder, run `bundle exec fastlane create_promo_screenshots`.
3. when the script ask to use the images in the `screenshots` folder, answer `y`.
4. wait for the script to complete and check that the promo screenshots have been generated in the `screenshot` folder.

Note: The script generates a `screenshots_orig` folder (git-ignored) with the source images. You can delete it when the process finishes.

### CODE REVIEW

File: Scripts/fastlane/ScreenshotFastfile [hunk 1]
```diff
@@ -0,0 +1,144 @@
+# This file contains the fastlane.tools configuration
```
Comment (frosty) [inline Scripts/fastlane/ScreenshotFastfile]: Can we remove these placeholder lines?

File: Scripts/fastlane/ScreenshotFastfile [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+  # Example:
+  # bundle exec fastlane download_promo_strings
+  #####################################################################################
+  desc "Downloads translated promo strings from GlotPress"
+  lane :download_promo_strings do |options| 
+    files = {
+      "app_store_screenshot-1" => {desc: "app_store_screenshot_2.txt"},
+      "app_store_screenshot-2" => {desc: "app_store_screenshot_5.txt"},
+      "app_store_screenshot-3" => {desc: "app_store_screenshot_3.txt"},
+      "app_store_screenshot-4" => {desc: "app_store_screenshot_1.txt"},
```
Comment (frosty) [inline Scripts/fastlane/ScreenshotFastfile]: Is it intentional that these numbers don't match up?

Comment (loremattei) [reply Scripts/fastlane/ScreenshotFastfile]: Yes, it is. 
The left value is the GlotPress key, so the translations get lost when it changes.
The right value is that target file name. The index in this name is related to the order of the screenshots in the Store. 

Screenshots used to have a certain order, then we changed it, but we didn't change the content of the messages, so the translations are still good. 
We use this hash to map every key to the right file (i.e. the screenshot it the right position).

**Comment by loremattei** [comment]
Thank you @frosty! 
I'm compiling a list of the translation issues we have with the app store metadata and I'm adding your findings to the list.

**Pull request merged**